### DISCUSSION TITLE
I was getting 503 in AWS Amplify in NEXT.JS Api Folder.

### DISCUSSION THREAD

**Discussion by HammadUllahKhan12** [vercel/next.js#40597] [Help] [answered]
I was getting 503 in AWS Amplify in NEXT.JS Api Folder.
![image](https://user-images.githubusercontent.com/48718737/190492410-4ce6d7e7-1e67-43ed-88c4-c532cd8e6c4a.png)

**Comment by HammadUllahKhan12** [comment, marked as answer]
I fixed the issue. The issue is you need to downgrade your next version to 11.3.3 and make sure the AWS Amplify is not getting the latest Next js version. 
You can set the NExt Version in AWS Amplify -> Build settings -> Edit


![image](https://user-images.githubusercontent.com/48718737/191397951-2985e6a5-7dca-4f63-a6b4-14e7ec2b0def.png)

**Comment by JohnDotOwl** [comment]
So if i want to use Next.js version 16, i would have to change the way i call APIs ? Probably hosted elsewhere ?
